src/test/java/app/bpartners/api/unit/repository/AreaPictureAnnotationRepositoryTest.java

```diff
@@ -0,0 +1,91 @@
+package app.bpartners.api.unit.repository;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import app.bpartners.api.model.AreaPictureAnnotation;
+import app.bpartners.api.repository.AreaPictureAnnotationRepository;
+import app.bpartners.api.repository.implementation.AreaPictureAnnotationRepositoryImpl;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.data.domain.Pageable;
+
+class AreaPictureAnnotationRepositoryTest {
+
+  @Mock private AreaPictureAnnotationRepository areaPictureAnnotationRepository;
+
+  @InjectMocks private AreaPictureAnnotationRepositoryImpl areaPictureAnnotationRepositoryImpl;
+
+  private AreaPictureAnnotation areaPictureAnnotation;
+  private Pageable pageable;
+  private String idUser = "user1";
+  private String idAreaPicture = "areaPic1";
+  private String annotationId = "annotation1";
+  private String invalidId = "invalid";
+
+  @BeforeEach
+  void setUp() {
+    MockitoAnnotations.openMocks(this);
+    areaPictureAnnotation =
+        AreaPictureAnnotation.builder()
+            .id(annotationId)
+            .creationDatetime(Instant.now())
+            .idUser(idUser)
+            .idAreaPicture(idAreaPicture)
+            .annotationInstances(List.of())
+            .build();
+
+    pageable = Pageable.unpaged();
+  }
+
+  @Test
+  void findAllBy_returnsAnnotationList() {
```

Comment:
Be consistent when you are naming your method. Choose between `CamelCase` and `snake_case`